sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -363,6 +364,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowWebPropertyBagUpdateWhenDenyAddAndCustomizePagesIsEnabled
+Enables or disables web property bag update when DenyAddAndCustomizePages is enabled. When AllowWebPropertyBagUpdateWhenDenyAddAndCustomizePagesIsEnabled is set to $true, web property bag can be updated even if DenyAddAndCustomizePages is turned on when the user had AddAndCustomizePages (prior to DenyAddAndCustomizePages removing it).
```

Comment:
>DenyAddAndCustomizePages is turned on when the user had AddAndCustomizePages (prior to DenyAddAndCustomizePages removing it) [](http://example.com/codeflow?start=211&length=124)

this part of the sentence:
> DenyAddAndCustomizePages is turned on when the user had AddAndCustomizePages (prior to DenyAddAndCustomizePages removing it)

is very confusing -- I've read it multiple times and still can't make sense of it. please revise it for clarity.